### ISSUE TITLE
No response from providers

### ISSUE THREAD

**Issue by SrdjanMilic** [JimmyLaurent/torrent-search-api#97]
It seems that all providers except "Rargb" are death or malfunctioned.
For some of them, i could say for quite a while.

Couple of days ago, I've checked all providers with provided api method: torrentSearch.isProviderActive('provider_name');

Response was `true` only from "Rargb".

**Comment by sayem314** [comment]
When I first discovered torrent-search-api and tried to use it and it failed for maximum providers so I have created my own and have been using them in torrenter. You can take a look at [torrent-indexer](https://github.com/sayem314/torrent-indexer) it is also lightweight and faster. I understand torrent-indexer is not well documented but take a look at the example file, it's really simple to use.

[compare](https://npmcompare.com/compare/torrent-indexer,torrent-search-api)

**Comment by JimmyLaurent** [comment]
Sorry guys, I didn't update the project for a while now. It's time consuming to keep up with the changes on torrents websites.

But don't worry I got you and I'm working on the next release, you can have a look at this [branch](https://github.com/JimmyLaurent/torrent-search-api/tree/fix-and-upgrade).
I upgraded dependencies, fixed some of the broken providers and added new ones (yts, limetorrents, eztv). I should publish by this end of the week.


@SrdjanMilic `isProviderActive` returns `true` when you have enabled the provider.

```js
// Disable provider
TorrentSearchApi.disableProvider('provider_name');
const result = torrentSearch.isProviderActive('provider_name');
// result => false

// Enable provider
TorrentSearchApi.enableProvider('provider_name');
const result = torrentSearch.isProviderActive('provider_name');
// result => true
```

**Comment by JimmyLaurent** [comment]
Providers have been fixed in the last 2.1.0 release.
